README.md

```diff
@@ -9,26 +9,35 @@
+## How does Rehearsal help with migration?
 
-Rehearsal Migrate can be executed both manually and automated. It is capable of maintaining proper migration order from leaf to trunk. It is generic enough to allow for the migration of JS to TS for all web applications. It allows for both multi-pass and single-pass migration processes. It provides industry standard, type inference wherever possible and provides the ability to monitor micro migration steps with macro insights. Once migrated to TypeScript, Rehearsal tasks enable a manual type tighten for strictness. After all tasks have been completed, Rehearsal Upgrade takes over. All in all, Rehearsal Migrate will increase your productivity when migrating a given JavaScript project to TypeScript.
+Rehearsal is capable of maintaining proper migration order from leaf to trunk. It is generic enough to allow for the migration of JS to TS for all web applications. It allows for both multi-pass and single-pass migration processes. It provides industry standard, type inference wherever possible and provides the ability to monitor micro migration steps with macro insights.
 
-## Upgrade
+Once your source code is moved to TypeScript, Rehearsal will fix as many errors as possible. Unresolved errors will be annotated and surpressed with with a [Rehearsal TODO](#what-is-a-rehearsal-todo). This allows for safe iteration, to a strictly typed project.
 
-Rehearsal Upgrade can be executed both manually and automated. Rehearsal creates a system which is capable of testing nightly and beta releases of TypeScript so that you can receive early signals on potential breaking changes in the TypeScript compiler. Additionally, Rehearsal can autofix type errors flagged by the compiler. It tests your TypeScript project against a newer version of TypeScript, transforms the code to align it with the new TypeScript version and provides Rehearsal tasks which enable a manual type tighten for strictness.
+## How does Rehearsal help with upgrading versions of TypeScript?
 
-# CLI
+Rehearsal can be executed both manually and automated. Rehearsal creates a system which is capable of testing nightly and beta releases of TypeScript so that you can receive early signals on potential breaking changes in the TypeScript compiler. Additionally, Rehearsal can autofix type errors flagged by the compiler. It tests your TypeScript project against a newer version of TypeScript, transforms the code to align it with the new TypeScript version and provides Rehearsal tasks which enable a manual type tighten for strictness.
 
-## Installation
+# Setup
 
-Add `@rehearsal/cli` to your project as a **devDepndency**.
+Ensure your project has the following dependencies:
+
+TBD
+
+# Installation
+
+Add `@rehearsal/cli` to your project as a **devDependency**.
 
 ```bash
 yarn add -D @rehearsal/cli
 # or
 pnpm add -D @rehearsal/cli
 ```
 
+Note: `@rehearsal/cli` must be installed as a devDependency as it requires your project's version of TypeScript to run.
```

Comment:
Why this? It doesn't need to use the project's version to run, but it will be using a project version of TS to compile a Program (LanguageService).

Reply:
Yes it does. How typescript is resolved is relative to the project's installed version of typescript. If `@rehearsal/cli` is installed globally or with volta install it will not work.

Reply:
> If @rehearsal/cli is installed globally or with volta install it will not work.

Installed rehearsal is a bunch of JS files that use TS for some work.
If you install @rehearsal/cli globally (as I always do) it uses the closest to the target project TypeScript to do the work. 
If the project has TS installed as deps / devDeps - it will use it, if the project doesn't have TS installed - it will try to use global (which is not rehearsal issue, but project configuration).

Reply:
Oh... seems it used to work, but broken now... 

```
Cannot find package 'typescript' imported from /Users/~/.volta/tools/image/packages/@rehearsal/cli/lib/node_modules/@rehearsal/cli/node_modules/@rehearsal/migrate/dist/src/migrate.js
```

weird